manifests/params.pp

```diff
@@ -8,66 +8,58 @@
   #   the most specific declaration of proxy.
   $proxy = 'absent'
 
-  if $facts['os']['name'] == 'Amazon' and $facts['os']['release']['major'] == '2' {
-    # Amazon Linux 2 is equivalent of Enterprise Linux 7 so we use that version for epel
-    # https://aws.amazon.com/premiumsupport/knowledge-center/ec2-enable-epel/
-    $os_maj_release = '7'
-  } else {
-    $os_maj_release = $facts['os']['release']['major']
```

Comment:
Looks like this variable is still used in places and breaking the tests. I wouldn't mind keeping it in either to keep the patch smaller.